warehouse/static/js/warehouse/controllers/horizontal_tabs_controller.js

```diff
@@ -19,6 +19,14 @@ export default class extends Controller {
 
 
   initialize() {
+    // Select the tab with errors if there are any
+    this.tabPanelTargets.forEach((target, index) => {
+      const aElement = target.querySelector("div#errors");
+      if (aElement) {
+        this.index = index;
+      }
+    });
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Should this be accompanied by a js test?

Reply:
Grumble grumble, ac4f804.